package.json

```diff
@@ -2,16 +2,30 @@
   "name": "@primer/behaviors",
   "version": "1.0.2",
   "description": "Shared behaviors for JavaScript components",
-  "main": "./dist/index.js",
-  "type": "module",
-  "types": "dist/index.d.ts",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "require": "./dist/cjs/index.js",
+      "module": "./dist/esm/index.js"
+    },
+    "./utils": {
+      "types": "./dist/utils/index.d.ts",
+      "require": "./dist/cjs/utils/index.js",
+      "module": "./dist/esm/utils/index.js"
```

Comment:
I saw there were many options we can provide here including `import`, `default`, `node`. How did you choose these two?

Reply:
I'm trying to remember where exactly I pulled these from, but there are many sources that don't all give the same keys.  https://nodejs.org/api/packages.html#conditional-exports is a good source, and it lines up more with the values you gave.  We should probably switch to `node` and `default`